src/components/footer/footer.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { FC, ReactNode } from 'react'
+import classNames from 'classnames'
+import { NativeProps, withNativeProps } from '../../utils/native-props'
+import { mergeProps } from '../../utils/with-default-props'
+import { Divider } from '../divider/divider'
+import Tag, { TagProps } from '../tag'
+
+const classPrefix = `adm-footer`
+
+export type LinkItem = {
+  label: string
+  value: string
+}
+
+export type ChipItem = {
+  text: string | ReactNode
+  tagProps?: TagProps
+  type?: 'plain' | 'link'
+}
+
+export type FooterProps = {
+  label?: string | ReactNode
+  links?: LinkItem[]
+  content?: string | ReactNode
+  chips?: ChipItem[]
+  onChipClick?: (item: ReactNode | any, index: number) => void
```

Comment:
```suggestion
  onChipClick?: (item: ChipItem, index: number) => void
```
感觉直接这样就行啦？为什么要联合 `any`? 下面的 `onLinkClick` 也是一样~